docs/reference/setup/cluster_restart.asciidoc

```diff
@@ -30,55 +33,108 @@ PUT _cluster/settings
 // TEST[skip:indexes don't assign]
 --
 
-. *Perform a synced flush*
+. *Stop indexing and perform a synced flush.*
 +
 --
 
-Shard recovery will be much faster if you stop indexing and issue a
-<<indices-synced-flush, synced-flush>> request:
+Performing a <<indices-synced-flush, synced-flush>> speeds up shard
+recovery.
 
 [source,sh]
 --------------------------------------------------
 POST _flush/synced
 --------------------------------------------------
 // CONSOLE
 
-A synced flush request is a ``best effort'' operation. It will fail if there
-are any pending indexing operations, but it is safe to reissue the request
-multiple times if necessary.
+Synced flush operations are performed on a "best effort" basis. The request
+fails if there are any pending indexing operations, but it is safe to
```

Comment:
This request bothers me a bit. That "best effort" bit means that it'll return a `200 OK` response but include a list of failures. I think we should make it super clear that you have to check the response to make sure it doesn't list any failures.